src/NuGet.Core/NuGet.Configuration/PackageNamespaces/PackageNamespacesProvider.cs

```diff
@@ -1,62 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-
-namespace NuGet.Configuration
-{
-    internal class PackageNamespacesProvider
```

Comment:
I meant to rename this file but I forgot what user actions in the IDE lead to deleting this file and creating a new one with the name `PackageSourceMappingProvider.cs`